store/auth.go

```diff
@@ -341,8 +295,8 @@ func requestDeviceSession(serialAssertion, sessionRequest, previousSession strin
 		headers["X-Device-Authorization"] = fmt.Sprintf(`Macaroon root="%s"`, previousSession)
 	}
 
-	_, err = retryPostRequest(MyAppsDeviceSessionAPI, headers, deviceJSONData, func(ok bool, resp *http.Response) error {
-		if ok {
+	_, err = retryPostRequest(MyAppsDeviceSessionAPI, headers, deviceJSONData, func(resp *http.Response) error {
+		if resp.StatusCode == 200 || resp.StatusCode == 202 {
```

Comment:
It would make sense to use http.StatusOK and http.StatusCreated constants here for consistency.